fe/src/components/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Flex,
+  Box,
+  Image,
+  useColorModeValue,
+  Icon,
+  chakra,
+  Tooltip,
+} from '@chakra-ui/react';
+import {
+  AddIcon,
+  CheckIcon,
+  CloseIcon,
+  HamburgerIcon,
+} from '@chakra-ui/icons';
+
+// @todo - Remove this interface after we have a type for Product as "single source of truth".
+interface Product {
```

Comment:
Currently it may seem like the ProductCard only needs a product for props, but it needs the product + the props for the element itself.

I say keep this interface (which is temporary as we'll be getting the types from autogenerated gql stuff anyways)
and simply add something like this: 
```ts
interface ProductCardProps {
  product: Product
  fullWidth: bool
  QuantityOfProducts: integer
  etc...
}
```
and then use it in the react element

Reply:
I think im not completely sure what are the functionalities for element props and why they're needed. 🤔  What do you mean exactly?

Reply:
Those were just some examples of props we might have in the future. For example if the school has 20x of some product it would make sense to show it as a single product that the school has 20 of, instead of just showing 20 cameras in a row.

Reply:
Okay, i will add some start up interface for that.